File: justmeetbackend/src/main/java/it/unicam/justmeetbackend/classi/Comune.java
package it.unicam.justmeetbackend.classi;

public class Comune {
    
    String nome;
    String siglaProvincia;

    public Comune(String nome, String siglaProvincia) {
        this.nome = nome;
        this.siglaProvincia = siglaProvincia;
    }

    public Comune() {}

    /**
     * @return the nome
     */
    public String getNome() {
        return nome;
    }

    /**
     * @param nome the nome to set
     */
    public void setNome(String nome) {
        this.nome = nome;
    }

    /**
     * @return the provincia
     */
    public String getSiglaProvincia() {
        return siglaProvincia;
    }

    /**
     * @param provincia the provincia to set
     */
    public void setSiglaProvincia(String provincia) {
        this.siglaProvincia = provincia;
    }

    
}

File: justmeetfrontend/lib/model/comune.dart
class Comune {
  String nome;
  String siglaProvincia;
  Comune(this.nome, this.siglaProvincia);
  Comune.fromjson(Map<String, dynamic> json)
  :
    this.nome = json['nome'],
    this.siglaProvincia = json['siglaProvincia'];
}